src/ProjectTemplates/Web.ItemTemplates/content/MvcController/.template.config/template.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "MVC Controller",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A MVC Controller with or without read/write actions",
```

Comment:
Drop the leading "a" to make it `MVC Controller with or without read/write actions`